src/openms/source/ANALYSIS/OPENSWATH/MRMAssay.cpp

```diff
@@ -185,9 +185,9 @@ namespace OpenMS
     OpenMS::MRMIonSeries mrmis;
 
     size_t j = 0;
-    for (size_t i = 0; i < exp.getTransitions().size(); ++i)
+    for (Size tr_it = 0; tr_it < exp.getTransitions().size(); ++tr_it)
```

Comment:
could you find another name than *_it (like 'j' or 'k')?!
It suggests this is an iterator, which it is not.